packages/stream-tile/src/tile-document.ts

```diff
@@ -355,7 +361,7 @@ export class TileDocument<T = Record<string, any>> extends Stream {
       prev: this.tip,
       id: this.state.log[0].cid,
     }
-    return _signDagJWS(signer, commit)
```

Comment:
@willex what is a rationale behind this?

Reply:
Creation of the helper function `_makeRawCommit` with breakout from `makeCommit` was necessary as to write a test case. It would otherwise not able to update the `metaData` after the content has been signed to verify to side step the `tile-controller` input verification and be able to trigger the `tile-controller-handler` check instead.

Reply:
IMO, it is better to use old fashioned functions.

Reply:
This _signDagJWS helper is an implementation detail of `makeCommit()`, it shouldn't be considered part of the public API.  The only reason we need it exposed at all is for a unit test.  I don't think exporting it as its own top-level function makes sense for something that should never be called directly outside of this file or its corresponding unit test